core/src/commonMain/kotlin/com/powersync/db/Queries.kt

```diff
@@ -59,65 +84,91 @@ public interface Queries {
     ): RowType?
 
     /**
-     * Execute a read-only (SELECT) query every time the source tables are modified and return the results as a List in [Flow].
+     * Returns a [Flow] that emits whenever the source tables are modified.
+     *
+     * @param tables The set of tables to monitor for changes.
+     * @param throttleMs The minimum interval, in milliseconds, between queries. Defaults to null.
```

Comment:
This doesn't actually run a query.

I think it would also be good to mention what happens if the database updates more frequently than the `throttleMs`. IIRC we collect writes into a growing set that is them emitted later?

"Defaults to null" is also misleading since the implementation does apply a default (`DEFAULT_WATCH_THROTTLE`) internally when no value is given. From that sentence, I would assume that no throttling takes place when the default value of `null` is passed.

Reply:
Good point! This implementation actually wasn't accumulating the set of changed tables between throttles - an event would have been emitted, but value of the tables could be incorrect. I've updated the implementation to accumulate the changed tables.

I've updated the defaults for `throttleMs` and explained the behaviour in the comments.